src/server/scripts/Northrend/ChamberOfAspects/RubySanctum/boss_halion.cpp

```diff
@@ -346,7 +346,7 @@ class boss_halion : public CreatureScript
                             events.ScheduleEvent(EVENT_BREATH, randtime(Seconds(16), Seconds(25)));
                             break;
                         case EVENT_ACTIVATE_FIREWALL:
-                            // Flame ring is activated 5 seconds after starting encounter, DOOR_TYPE_ROOM is only instant.
+                            // Flame ring is activated 5 seconds after starting encounter, EncounterStateForOpenDoor::NotInProgress is only instant.
```

Comment:
In retail, normally doors that close in retail for a boss fight take 5s to close. Why is this any different?

Reply:
out of scope